websrc/source/documents/networking/l2-vlan.md

```diff
@@ -78,16 +78,16 @@ Log in to containers and verify the IP address has been allocated from the netwo
 ```
 docker ps
 CONTAINER ID        IMAGE                          COMMAND             CREATED             STATUS              PORTS               NAMES
-049a85ec0c8c        alpine                         "sh"                7 minutes ago       Up 7 minutes                            contiv-demo-swarm-l2-4/desperate_bose
-f864841d626d        alpine                         "sh"                8 minutes ago       Up 8 minutes                            contiv-demo-swarm-l2-6/agitated_brahmagupta
-9f3aed39e95b        alpine                         "sh"                9 minutes ago       Up 8 minutes                            contiv-demo-swarm-l2-6/happy_pasteur
-a3111bb0c9e1        quay.io/coreos/etcd:v2.3.1     "/etcd"             10 minutes ago      Up 10 minutes                           contiv-demo-swarm-l2-4/etcd
-6881c4c1b380        quay.io/coreos/etcd:v2.3.1     "/etcd"             10 minutes ago      Up 10 minutes                           contiv-demo-swarm-l2-6/etcd
-a8eb4c23c009        quay.io/coreos/etcd:v2.3.1     "/etcd"             10 minutes ago      Up 10 minutes                           contiv-demo-swarm-l2-5/etcd
-b54850f408d8        skynetservices/skydns:latest   "/skydns"           5 weeks ago         Up 10 minutes       53/tcp, 53/udp      contiv-demo-swarm-l2-6/defaultdns
-b54850f408d8        skynetservices/skydns:latest   "/skydns"           5 weeks ago         Up 10 minutes       53/tcp, 53/udp      contiv-demo-swarm-l2-4/defaultdns
-b54850f408d8        skynetservices/skydns:latest   "/skydns"           5 weeks ago         Up 10 minutes       53/tcp, 53/udp      contiv-demo-swarm-l2-5/defaultdns
-[admin@contiv-demo-swarm-l2-4 ~]$ docker exec -it 049a85ec0c8c sh
+049a85ec0c8c        alpine                         "sh"                7 minutes ago       Up 7 minutes                            contiv-demo-legacy-swarm-l2-4/desperate_bose
+f864841d626d        alpine                         "sh"                8 minutes ago       Up 8 minutes                            contiv-demo-legacy-swarm-l2-6/agitated_brahmagupta
+9f3aed39e95b        alpine                         "sh"                9 minutes ago       Up 8 minutes                            contiv-demo-legacy-swarm-l2-6/happy_pasteur
+a3111bb0c9e1        quay.io/coreos/etcd:v2.3.1     "/etcd"             10 minutes ago      Up 10 minutes                           contiv-demo-legacy-swarm-l2-4/etcd
+6881c4c1b380        quay.io/coreos/etcd:v2.3.1     "/etcd"             10 minutes ago      Up 10 minutes                           contiv-demo-legacy-swarm-l2-6/etcd
+a8eb4c23c009        quay.io/coreos/etcd:v2.3.1     "/etcd"             10 minutes ago      Up 10 minutes                           contiv-demo-legacy-swarm-l2-5/etcd
+b54850f408d8        skynetservices/skydns:latest   "/skydns"           5 weeks ago         Up 10 minutes       53/tcp, 53/udp      contiv-demo-legacy-swarm-l2-6/defaultdns
```

Comment:
I have a feeling that these skydns containers are bogus as well.

Why don't you leave out the documents outside of tutorials for now. We can fix those in another PR.